Libplanet/Action/AccountStateDeltaImpl.cs

```diff
@@ -300,23 +300,13 @@ public virtual IAccountStateDelta BurnAsset(Address owner, FungibleAssetValue va
 
         /// <inheritdoc/>
         [Pure]
-        public IAccountStateDelta SetValidator(PublicKey validatorKey, BigInteger power)
+        public IAccountStateDelta SetValidator(Validator validator)
         {
-            var validator = new Validator(validatorKey, power);
-            var validators = GetValidatorSet().Validators.Remove(validator);
-
-            if (power.Sign < 0)
-            {
-                throw new ArgumentOutOfRangeException(
-                    nameof(power),
-                    "The field \"power\" cannot be negative.");
-            }
-            else if (power.Sign == 0)
-            {
-                return UpdateValidatorSet(new ValidatorSet(validators.ToList()));
-            }
-
-            return UpdateValidatorSet(new ValidatorSet(validators.Add(validator).ToList()));
+            Log.Debug(
+                "Update validator set with validator {PublicKey} {Power}",
+                validator.PublicKey,
+                validator.Power);
```

Comment:
I think this log was used for testing only.

Reply:
Resolved with commit amending